index.js

```diff
@@ -1497,6 +1497,25 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# both :: (a -> Boolean) -> (a -> Boolean) -> a -> Boolean
+  //.
+  //. Takes two unary predicates and a value of any type, and returns
+  //. true if the value satisfies both predicates; false otherwise.
```

Comment:
Let's wrap "true" and "false" with backticks so they are rendered as `true` and `false`.